LinkedList/sortedListToBST.swift

```diff
@@ -0,0 +1,57 @@
+/**
+ * Question Link: https://leetcode.com/problems/convert-sorted-list-to-binary-search-tree/
+ * Primary idea: Convert the linked list to array, handle array using Divide and Conqure
+ * Time Complexity Per Action: O(N), Space Complexity: O(N)
+ *
+ */
+
+ /**
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     public var val: Int
+ *     public var next: ListNode?
+ *     public init(_ val: Int) {
+ *         self.val = val
+ *         self.next = nil
+ *     }
+ * }
+ */
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     public var val: Int
+ *     public var left: TreeNode?
+ *     public var right: TreeNode?
+ *     public init(_ val: Int) {
+ *         self.val = val
+ *         self.left = nil
+ *         self.right = nil
+ *     }
+ * }
+ */
+class Solution {
+    func sortedListToBST(_ head: ListNode?) -> TreeNode? {
+        var sortedArr: [Int] = []
+
+        var cur = head
+        while cur != nil {
+            sortedArr.append(cur!.val)
+            cur = cur!.next
+        }
```

Comment:
Do you think you can do something like this below, so you don't have to implicitly unwrap the variable which is not recommended in production code?

```
        while let newCurr = cur {
            sortedArr.append(newCurr.val)
            cur = newCurr.next
        }
```